crypto/keywrap/keywrap.c

```diff
@@ -0,0 +1,309 @@
+/* Example showing ECC shared secret and HKDF to generate a symmetric key */
+/*
+gcc -Wall -okeywrap -lwolfssl keywrap.c
```

Comment:
I gave this a try and needed to have -lwolfssl after keywrap.c. Like this `gcc -I/usr/local/include -Wall -okeywrap -L/usr/local/lib  keywrap.c -lwolfssl`